llvm/test/tools/llvm-readobj/ELF/bb-addr-map-pgo-analysis-map.test

```diff
@@ -0,0 +1,221 @@
+## This test checks how we handle PGO Analysis Map in --bb-addr-map.
+
+## Fails on windows (https://github.com/llvm/llvm-project/issues/60013).
+# UNSUPPORTED: system-windows
```

Comment:
Remove this.

Reply:
Good catch, think I did this before the fix.